src/app/components/CheckInputArea/CheckInputArea.tsx

```diff
@@ -2,7 +2,7 @@ import styled from 'styled-components';
 import DefaultButton from '../ButtonDefault/ButtonDefault';
 
 const CheckInputArea = styled.form`
-  background: var(--background-color-tertiary);
```

Comment:
another common naming convention is something like --color-neutral-200 up to 900